### ISSUE TITLE
Unable to log in to D365BC after password change

### ISSUE THREAD

**Issue by chris-amco** [Microsoft/navcontainerhelper#206]
I have a docker container which I have been using for Business Central extension development for the past 2-3 weeks. After changing the password of my Windows account this morning, I was no longer able to log in to Business Central. After trying to log in 3 times, this message is shown:
```
401 - Unauthorized: Access is denied due to invalid credentials.
You do not have permission to view this directory or page using the credentials that you supplied.
```
And when I try to publish an extension from VSCode, I get:
```
[2018-09-24 11:42:05.41] Sending request to http://d365bc:7049/nav/dev/apps?SchemaUpdateMode=recreate
[2018-09-24 11:42:07.86] The request for path /nav/dev/apps?SchemaUpdateMode=recreate failed with code Unauthorized. Reason: Unauthorized
```

The container is running normally and I am able to log in to BC with another account. However this doesn't help much as I still need to be able to log in with my usual account in order to publish extensions from VSCode.

I found that I was able to reproduce the problem by changing the password of the account that was able to log in. After changing it, I was unable to log in, and after changing it back to the old password, I was able to log in again. That was a local account but my usual account is an Active Directory domain user, and I cannot go back to the old password due to the domain's password policy. 

I've tried restarting the container, restarting docker and restarting the host computer, none of which helped.

Is there a way to reset the trust relationship between the container user and the host user? Or recreate the container user? Or should I just create a new container?

Thanks in advance.

**Comment by chris-amco** [comment]
Thanks a lot for your help. 

It looks like changing the password would have worked, but I made the silly mistake of recreating the user instead, after which I was able to access the BC web site but unable to log in to BC itself. And I was unable to recreate the user on BC because user names must be unique and existing users may not be deleted if they have ever logged in. 

So I guess I'll just recreate the container. But next time it happens I will just change the password in the container.
